### PR TITLE
[WC-1124] Docs for new charts module

### PR DESCRIPTION
**Pull request by iobuhov**
In this PR we adding new docs page for upcoming charts module.
This module is not yet released, but we working hard on publishing it to marketplace. Please, reach someone in Web Content team to align merge/release dates.

### CODE REVIEW

**Comment by Keraito** [comment]
@iobuhov right now your PR descriptions mention that only 2 pages were done, can we already do a tech review or should we wait?

File: content/en/docs/appstore/modules/charts/_index.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+This property controls horizontal and vertical rulers of the chart:
+
+- **None** — no rulers visible.
+- **Horizontal** — only horizontal ruler is visible.
+- **Vertical** — only vertical ruler is visible.
+- **Both** — both types of rulers are visible.
+
+
+
+### Dimensions
```
Comment (Keraito) [inline content/en/docs/appstore/modules/charts/_index.md]: @ConnorLand this section is basically the same as https://github.com/mendix/docs/blob/development/content/en/docs/appstore/widgets/maps.md#36-dimensions. So maybe it's better to just copy it as it also explains the relationships between the different properties.

File: content/en/docs/appstore/modules/charts/_index.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+This property allows you to save your custom **Configuration** settings for this widget.
+
+This object will be merged with default settings and passed to underlying [plotly javascript](https://plotly.com/javascript/) library. To see available settings and examples check [Configuration Options in JavaScript](https://plotly.com/javascript/configuration-options/) section on plotly docs website.
+
+{{< figure src="/attachments/appstore/modules/charts/custom-config.png" alt="Settings dialog window with Advanced tab being active. Tab includes two text area on of which is focused." >}}
+
+
+
+##
```
Comment (Keraito) [inline content/en/docs/appstore/modules/charts/_index.md]: @iobuhov you're missing a title here

File: content/en/docs/appstore/modules/charts/_index.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+#### Slice color
+
+Optional expression that when evaluated will return uniq color for single series.
+
+#### Hole radius
+
+Controls hole radius at the center of the chart. When value is zero the hole is not visible.
+
+## Migration from Charts v1 and v2 to Charts Module
```
Comment (Keraito) [inline content/en/docs/appstore/modules/charts/_index.md]: Just to make sure: we're going from v2 to v3. The new ones are v3.

File: content/en/docs/appstore/modules/charts/_index.md [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+**Note**: if you using AnyChart widget in your project we highly recommend you to migrate your charts to new charts module — it has better developer experience and give you advanced set of chart widgets to visualize your data.
+
+{{% /alert %}}
+
+
+### Step 3 - Installing Charts module from marketplace
+
+So, now you can open Studio Pro, go to marketplace and install brand new Charts module.
+
+<!-- Add screenshots of marketplace with Charts module -->
```
Comment (Keraito) [inline content/en/docs/appstore/modules/charts/_index.md]: @iobuhov are you gonna pick this up?

**Review by Keraito — COMMENTED** [review]
Generally looks good, very nice screenshots. Have minor remarks/questions. After addressing those, the docs team can take this as I think the structure (and grammar) can be optimized a bit cc @ConnorLand

File: content/en/docs/appstore/modules/charts/_index.md (hunk 4 above)
Comment (iobuhov) [reply content/en/docs/appstore/modules/charts/_index.md]: No, I'll remove it

**Comment by iobuhov** [comment]
@ConnorLand, ready for your review.

**Review by ConnorLand — CHANGES_REQUESTED** [review]
Almost done. I just need to do a final check on references to "version 3" and references to the "new widget" (must be changed to "new module").

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**